doc/conf.py

```diff
@@ -22,6 +22,9 @@
 from datetime import datetime
 import time
 
+# Release mode enables optimizations and other related options.
+_doc_release_mode = tags.has('release')  # noqa
```

Comment:
```suggestion
is_release_build = tags.has('release')  # noqa
```
(or even `is_release`) because that reads better below. We don't nee to mark this as private. The conf is not public anyway.

Reply:
I didn't want people to think it could be set by `sphinx-build -D`, though it's not possible to actually use anyway without being registered in `setup`.